README.md

```diff
@@ -4,6 +4,10 @@ This is a Jekyll site. It is a simple site to showcase the work of an organizati
 
 This particular site is made for use with the [Federalist](https://github.com/18f/federalist) in mind.
 
+This template uses the CSS and JavaScript from [U.S. Web Design System](https://standards.18f.gov). To update the version of the system, download the code from the website, then copy the CSS file to `assets/css/`. The file must include the version number, like this: `uswds.v1.1.0.css`. Copy the minified JavaScript file to `assets/js/lib/`. Again, rename the file to use the version number, so it looks like: `uswds.v1.1.0.min.js`.
+
+In your `_config.yml`, update the `wds_version` value to the new version number (this should match the number following the `v` in the filenames you just created). 
+
```

Comment:
I'm curious for feedback on this approach. I could have just named the files `uswds.css`/`uswds.min.js`, and then the only thing necessary for updating would be overwriting those files with new ones. That approach has the downside of clobbering old files, though, which makes it potentially difficult to revert to old files if the new ones do unexpected things. It also makes it difficult to tell which version of the standards the template is running.